src/main/java/com/launchdarkly/client/FeatureFlag.java

```diff
@@ -61,33 +61,32 @@ JsonElement getOffVariationValue() {
     return null;
   }
 
-  EvalResult evaluate(LDUser user, FeatureStore featureStore) {
+  EvalResult evaluate(LDUser user, FeatureStore featureStore) throws EvaluationError {
```

Comment:
I realized the changes you made were to solve a specific problem, but If we're moving to this pattern of throwing exceptions during evaluation we should consider change it across the board for consistency. One example is Line 57 in this file: getOffVariationValue().